lib/ui/window/pointer_data_packet_converter_unittests.cc

```diff
@@ -129,6 +131,7 @@ void CreateSimulatedTrackpadGestureData(PointerData& data,  // NOLINT
   data.pan_delta_y = 0.0;
   data.scale = scale;
   data.rotation = rotation;
+  data.view_id = kImplicitViewId;
```

Comment:
Nit: Consider using `0` directly instead. Ideally the engine shouldn't make any assumptions about the ID of the implicit view.